### ISSUE TITLE
Adding an account

### ISSUE THREAD

**Issue by DrenIT** [chris124/iOSSocial#37]
.

**Comment by mrchristopher124** [comment]
Hi,

Sorry for the late response. One important thing to note is that your redirect URL should NOT point to a reachable endpoint. Can you try setting an endpoint that isn't reachable on the net (just come up with a dummy endpoint that doesn't exist that uses your domain: www.mydomain.com/dummycallback) and let me know? Make sure you also set this same endpoint in your settings in the Instagram developer site.

Hope this helps,
Chris

**Comment by DrenIT** [comment]
Hi,

thanks for your answer. I have already found the problem, it was a missing backslash. Now everything is good.
But one thing is missing, refreshing token. Is it right?

regards,
Asdren Gashi 

----- Ursprüngliche Mail -----
Von: "Christopher White" reply@reply.github.com
An: "DrenIT" register@drenit.de
Gesendet: Sonntag, 8. Juli 2012 17:54:09
Betreff: Re: [iOSSocial] Adding an account (#37)

Hi,

Sorry for the late response. One important thing to note is that your redirect URL should NOT point to a reachable endpoint. Can you try setting an endpoint that isn't reachable on the net (just come up with a dummy endpoint that doesn't exist that uses your domain: www.mydomain.com/dummycallback) and let me know? Make sure you also set this same endpoint in your settings in the Instagram developer site.

Hope this helps,
Chris

---

**Comment by mrchristopher124** [comment]
I'm not sure I follow the question. Can you rephrase or give more info?

On Sun, Jul 8, 2012 at 12:17 PM, DrenIT <
reply@reply.github.com

**Comment by DrenIT** [comment]
for example flickr:

The Token of Flickr will expire but i can't see that you did handle this case. 

----- Ursprüngliche Mail -----
Von: "Christopher White" reply@reply.github.com
An: "DrenIT" register@drenit.de
Gesendet: Sonntag, 8. Juli 2012 18:40:52
Betreff: Re: [iOSSocial] Adding an account (#37)

I'm not sure I follow the question. Can you rephrase or give more info?

On Sun, Jul 8, 2012 at 12:17 PM, DrenIT <
reply@reply.github.com

> wrote:
> 
> Hi,
> 
> thanks for your answer. I have already found the problem, it was a missing
> backslash. Now everything is good.
> But one thing is missing, refreshing token. Is it right?
> 
> regards,
> Asdren Gashi
> 
> ----- Ursprüngliche Mail -----
> Von: "Christopher White" reply@reply.github.com
> An: "DrenIT" register@drenit.de
> Gesendet: Sonntag, 8. Juli 2012 17:54:09
> Betreff: Re: [iOSSocial] Adding an account (#37)
> 
> Hi,
> 
> Sorry for the late response. One important thing to note is that your
> redirect URL should NOT point to a reachable endpoint. Can you try setting
> an endpoint that isn't reachable on the net (just come up with a dummy
> endpoint that doesn't exist that uses your domain:
> www.mydomain.com/dummycallback) and let me know? Make sure you also set
> this same endpoint in your settings in the Instagram developer site.
> 
> Hope this helps,
> Chris
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/chris124/iOSSocial/issues/37#issuecomment-6832181
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/chris124/iOSSocial/issues/37#issuecomment-6832345

---
